configs/authnz-frontend/nginx.conf.template

```diff
@@ -0,0 +1,45 @@
+server {
+  listen ${LISTEN_ADDRESS};
+
+  server_name ${SERVER_NAME};
+
+  root /usr/share/nginx/html;
+
+  index index.html;
+
+  location / {
+       try_files $uri $uri/ /index.html;
+  }
+
+  location ~ $/(static)/ {
+    gzip on;
+    gzip_http_version  1.0;
```

Comment:
I'm not sure what difference this would have, but why isn't this `1.1`?

Reply:
Will fix. Seems 1.1 is the default, so I'll remove this stanza